pages/03.community/02.chat_rooms/chat_rooms.es.md

```diff
@@ -10,37 +10,37 @@ routes:
 El proyecto YunoHost utiliza salas de chat como medio de communicación.
 
 Puede unirse a una sala de chat utilizando :
-- un [cliente IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) por ejemplo [KiwiIRC](https://kiwiirc.com/client/irc.freenode.net/yunohost)
+- un [cliente IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) por ejemplo [KiwiIRC](https://web.libera.chat/#yunohost)
```

Comment:
Let's maybe replace KiwiIRC with Libera.chat in the text too? Or let's leave it since that the software used by Libera.chat anyways?

Reply:
idk honestly :/

what would be the most understandable for users?

Reply:
Let's keep it that way. If people complain, we will know soon enough. :p